ethereum/api/src/test/resources/org/hyperledger/besu/ethereum/api/jsonrpc/debug/debug_accountAt.json

```diff
@@ -6,16 +6,16 @@
     "params": [
       "0xc8df1f061abb4d0c107b2b1a794ade8780b3120e681f723fe55a7be586d95ba6",
       0,
-      "0xbcde5374fce5edbc8e2a8697c15331677e6ebf0b"
```

Comment:
changed this test fixture temporarily to access an account available in the updater.

Reply:
reverted